common/txmgr/resender.go

```diff
@@ -140,8 +140,7 @@ func (er *Resender[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) resendUnco
 		return fmt.Errorf("Resender failed getting enabled keys for chain %s: %w", er.chainID.String(), err)
 	}
 
-	// Tracker currently disabled for BCI-2638; refactor required
-	// resendAddresses = append(resendAddresses, er.tracker.GetAbandonedAddresses()...)
+	resendAddresses = append(resendAddresses, er.tracker.GetAbandonedAddresses()...)
```

Comment:
I'm thinking if this is a desired behaviour or not. If I abandon an address, do I want the TXM to retry for pending attempts? Why is this different compared to the confirmer? Abandoning an address to me seems more like "what happened happened, don't waste more time on that address". This might be a product-related question.

Reply:
Good question, I'll see what Prashant thinks since he initially suggested resending. I'm indifferent on whether or not we should resend. It's cool that we have the ability to do so now, but it's also fair to say that it's unnecessary.